pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -951,6 +954,45 @@ func (m *kubeGenericRuntimeManager) killPodWithSyncResult(pod *v1.Pod, runningPo
 	return
 }
 
+func (m *kubeGenericRuntimeManager) GeneratePodStatus(event *runtimeapi.ContainerEventResponse) (*kubecontainer.PodStatus, error) {
+	podIPs := m.determinePodSandboxIPs(event.PodSandboxMetadata.Namespace, event.PodSandboxMetadata.Name, event.PodSandboxStatus)
+
+	kubeContainerStatuses := []*kubecontainer.Status{}
+	for _, status := range event.ContainersStatuses {
+		cStatus := toKubeContainerStatus(status, m.runtimeName)
```

Comment:
this part can be replaced by `c.convertToKubeContainerStatus` right?

Reply:
Oh yes, thanks for pointing it out. :)

Reply:
Do we want to replace this part with convertToKubeContainerStatus?

Reply:
Yes, we would. Thanks for pointing it out. I will update the code.